### PR TITLE
Update AboutUs.md

### LINKED ISSUE #3: Update AboutUs page from default AB3 to CampusConnect
The AboutUs page needs to be updated from the default AboutUs from AB3 to one that's relevant to CampusConnect.

### PR DESCRIPTION
**Pull request by conradsoon**
Fixes #3

### CODE REVIEW

File: docs/AboutUs.md
Comment (lordidiot) [inline docs/AboutUs.md]: Please add `Scheduling and tracking` as well

**Review by lordidiot — CHANGES_REQUESTED** [review]
One minor change.
Also, should we add surnames for clarity? Doesn't have to be full names but maybe surnames would be good.

File: docs/AboutUs.md
Comment (conradsoon) [reply docs/AboutUs.md]: Have rectified the issues, please review again.

**Review by kokrui — APPROVED** [review]
LGTM, once this is merged to master I should be able to debug #5 as well

**Review by lordidiot — APPROVED** [review]
LGTM! Thanks for the quick fixes.

**Pull request merged**
